### DISCUSSION TITLE
Cloud scalability for query nodes

### DISCUSSION THREAD

**Discussion by chikymanzanares** [milvus-io/milvus#33739] [Q&A and General discussion]
Hello,

We have a Milvus database with 200 million vectors installed on Kubernetes. It uses a similarity model to detect duplicates. I would also like to use it for search, but we are not sure about scalability for handling many requests.
![Screenshot 2024-06-10 at 10 05 39](https://github.com/milvus-io/milvus/assets/32399434/9da14958-30ec-467a-9343-e5d96463ff37)

When we have a lot of requests, we manually change the number of query nodes in the deployment. Is there a way to automate this process instead of doing it manually?
![Screenshot 2024-06-10 at 10 07 18](https://github.com/milvus-io/milvus/assets/32399434/fc479e9b-e779-4d9b-b417-f801e1478290)

**Comment by yhmo** [comment]
So far as I know, we don't have plan to support auto-scale. Read this discussion for more info: https://github.com/milvus-io/milvus/discussions/25422

**Reply by chikymanzanares** [reply]
During peak search times, we experience an issue where a high volume of requests causes the search to go down. Do you have any suggestions or temporary solutions we can implement until official support is provided?
